### ISSUE TITLE
Line endings and .gitattributes

### ISSUE THREAD

**Issue by lol768** [korobi/Web#197]
We need a .gitattributes file to ensure consistent line ending handling. Aim: There should be no Windows-style line-endings in the repository at all.

Files that already have windows line endings need to be fixed:

@bionicrm please fix the file you committed:

```
➜  Web git:(master) ✗ grep -IUr --color "^M" --exclude-dir=vendor

web/assets/images/flags/america.svg:<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" 

web/assets/images/flags/america.svg:<polygon id="pt" points="-0.1624598481164531,0 0,-0.5 0.1624598481164531,0" transfo
web/assets/images/flags/america.svg:<g id="star"><use xlink:href="#pt" transform="rotate(-144)"/><use xlink:href="#pt" transform="rotate(-72)"/><use xlink:href="#pt"/><use xlink:href="#pt" transform="rotate(72)"/><use xlink:href="#pt" tra
web/assets/images/flags/america.svg:<g id="s5"><use xlink:href="#star" x="-0.252"/><use xlink:href="#star" x="-0.126"/>
web/assets/images/flags/america.svg:<g id="s6"><use xlink:href="#s5" x="-0.063"/><use xlink:href="#star" x="0.315"/></g
web/assets/images/flags/america.svg:<g id="x4"><use xlink:href="#s6"/><use xlink:href="#s5" y="0.054"/><use xlink:href=
web/assets/images/flags/america.svg:<g id="u"><use xlink:href="#x4" y="-0.216"/><use xlink:href="#x4"/><use xlink:href=


web/assets/images/flags/america.svg:<rect width="1235" height="650" fill="#FFF"/><use xlink:href="#stripe"/><use xlink:href="#stripe" y="100"/><use xlink:href="#stripe" y="200"/><use xlink:href="#stripe" y="300"/><use xlink:href="#stripe" y="400"/><use xlink:href="#stripe" y="500"/><use xlink:href="#stripe" y="600"/><rect width="494" height="350" fill="#3
➜  Web git:(master) ✗ 
```

**Comment by bionicrm** [comment]
Attempted fix in #198.

**Comment by lol768** [comment]
This issue has not been entirely fixed with the commit in `772d864` (we still want a .gitattributes file) though @bionicrm's file looks correct now.

**Comment by bendem** [comment]
it was, I added the .gitattribute file and git is clever enough to understand what's binary and what's not.
